libMBIN/Source/NMS/Globals/GcWaterGlobals.cs

```diff
@@ -19,7 +19,7 @@ public class GcWaterGlobals : NMSTemplate
         /* 0x024 */ public float FresnelBelowPow;
         /* 0x028 */ public float FresnelBelowMul;
         /* 0x02C */ public float FresnelBelowAlpha;
-        /* 0x030 */ public Colour WaterHeavyAirColour;
+        /* 0x030 */ public GcStyleProp_Colour WaterHeavyAirColour;
```

Comment:
I think this is still Just `Colour`. The exe shows it as such

Reply:
sorry, I don't remember from where I got that, I'll change it back to Colour